frontend/components/forms/fields/DropdownWrapper/_styles.scss

```diff
@@ -0,0 +1,12 @@
+// react-select's customStyles styles the internal components
```

Comment:
Can we make clear that `customStyles` is an object in the `.tsx`, and maybe add a link to
https://react-select.com/styles? Just so it's real clear where and how the styles are being defined